images/jenkins/config/template.apko.yaml

```diff
@@ -19,7 +14,7 @@ accounts:
 work-dir: /app
 
 entrypoint:
-  command: /usr/bin/java -Duser.home=/var/jenkins_home -jar /usr/share/java/jenkins/jenkins.war
+  command: jenkins.sh
```

Comment:
This comes from the `jenkins-entrypoint` package I think, so +1: https://github.com/wolfi-dev/os/blob/7b4049a1440c217c0b0e69cfced2e05deb6181d4/jenkins.yaml#L16